### PR TITLE
Fixing proxmox not showing the saved vmid

### PR DESCRIPTION
**Pull request by Turkidev**

### CODE REVIEW

**Comment by GJSBRT** [comment]
Why all the unnecessary formatting changes? Please only change actual code, it makes reviewing way easier

File: app/Extensions/Servers/Proxmox/Proxmox.php [hunk 1]
```diff
@@ -759,21 +771,25 @@ public function terminateServer($user, $parmas, $order, $product, $configurableO
...
+            throw new Exception('Unable to terminate server');
         return true;
     }
 
 
     public function getCustomPages($user, $parmas, $order, $product, $configurableOptions)
     {
         $vmType = $parmas['type'];
-        $vmid = $parmas['config']['vmid'];
+        $vmid = ExtensionHelper::getOrderProductConfig('vmid', $product->id);
```
Comment (GJSBRT) [inline app/Extensions/Servers/Proxmox/Proxmox.php:783]: Add a check if a value is returned

**Review by GJSBRT — CHANGES_REQUESTED** [review]

**Comment by Turkidev** [comment]
sorry its auto formatting in vscode

File: app/Extensions/Servers/Proxmox/Proxmox.php (hunk 1 above)
Comment (Turkidev) [reply app/Extensions/Servers/Proxmox/Proxmox.php:783]: check if its null?

Comment (GJSBRT) [reply app/Extensions/Servers/Proxmox/Proxmox.php:783]: Yes, In your helper function you return if there is no value, which should also be a `return null;` btw.

Comment (Turkidev) [reply app/Extensions/Servers/Proxmox/Proxmox.php:783]: if ($vmid == null) return;?

**Pull request closed without merging**